packages/framework/src/Model/Order/Item/OrderItemDataFactory.php

```diff
@@ -52,4 +73,30 @@ protected function addFieldsByOrderItemType(OrderItemData $orderItemData, OrderI
             $orderItemData->payment = $orderItem->getPayment();
         }
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Order\Item\OrderItemData $orderItemData
+     * @param \Shopsys\FrameworkBundle\Model\Order\Item\OrderItem $orderItem
+     * @return bool
+     */
+    protected function canUsePriceCalculation(OrderItemData $orderItemData, OrderItem $orderItem): bool
+    {
+        if ($this->orderItemPriceCalculation === null) {
+            return false;
```

Comment:
shouldn't this throw an exception?

Reply:
My original thinking was that it should just return `false` meaning it cannot decide whether it should use the price calculation. But this would definitely lead to unexpected behavior, especially because I use the property outside of this method as well. Addressed in 3a67de5d9